### PR TITLE
[backup] smoke test

### PR DESCRIPTION
**Pull request by msmouse**
## Motivation

Smoke test the DB backup toolchain.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Y

## Test Plan

` cargo x test -p testsuite -- --test smoke_test::test_db_restore --nocapture`

### CODE REVIEW

File: testsuite/tests/libratest/smoke_test.rs [hunk 1]
```diff
@@ -2045,3 +2050,219 @@ fn parse_waypoint(db_bootstrapper_output: &str) -> Waypoint {
...
+) -> f64 {
+    let now = Instant::now();
+    let mut transferred = 0f64;
+    while !quit.load(Ordering::Relaxed) {
+        client
+            .transfer_coins(&["tb", "0", "1", "1", "Coin1"], true)
+            .unwrap();
+        transferred += 1.0;
+
+        assert!(compare_balances(
```
Comment (lightmark) [inline testsuite/tests/libratest/smoke_test.rs:2253]: Does `transfer_coin` take effect immediately?

**Review by lightmark — APPROVED** [review]

File: testsuite/tests/libratest/smoke_test.rs (hunk 1 above)
Comment (msmouse) [reply testsuite/tests/libratest/smoke_test.rs:2253]: yea, the last parameter is "is_blocking" (think the cli)

**Pull request closed without merging**